src/Functions/map.cpp

```diff
@@ -147,6 +149,99 @@ class FunctionMap : public IFunction
     }
 };
 
+// mapFromArrays(keys, values) is a function that allows you to make key-value pair from a pair of arrays
+class FunctionMapFromArrays : public IFunction
+{
+public:
+    static constexpr auto name = "mapFromArrays";
+
+    static FunctionPtr create(ContextPtr)
```

Comment:
Feel free to write one-line functions actually on a single line ... it will produce better output when prepping for the function name.

```cpp
String getName() const override { return name; }
```